packages/message/src/StepFunctions.ts

```diff
@@ -92,8 +92,7 @@ export const pullStepFunctionEvent = async (
  *
  * @param {CMAMessage} stepMessage - Message for the step
  * @param {string} stepName - Name of the step
- * @returns {Promise<Object>} Parsed and updated event with target path replaced by remote message
- *
+ * @returns {Promise<object>} Parsed and updated event with target path replaced by remote message
```

Comment:
Can we be more specific? Not your change so won't hold up the PR.

Reply:
thanks